modules/wallet/defrag.go

```diff
@@ -130,10 +139,10 @@ func (w *Wallet) threadedDefragWallet() {
 		w.mu.Unlock()
 		return
 	}
+	w.mu.Unlock()
```

Comment:
this can change to an `RLock` now:
```go
w.mu.RLock()
unlocked := w.unlocked
w.mu.RUnlock()
if !unlocked {
    // Can't defrag if the wallet is locked.
    return
}
```